### ISSUE TITLE
Install RSync from `setup.sh`

### ISSUE THREAD

**Issue by jourdanrodrigues** [heydemoura/dotfiles#1]
Why not?

**Comment by heydemoura** [comment]
It can be helpful, but then you come to the challenge of supporting the variety of different package managers, and that can be tricky.

It's more reasonable to warn the user that it should have RSync installed. Don't you think?

**Comment by jourdanrodrigues** [comment]
I don't because, it's a requirement for your environment and, as I see it, there's no point in having a script to setup your environment if you still have to run things manually. It would be as easy as the following:

```bash
if [ -z "$(command -v rsync)" ]; then
    apt-get install rsync
fi
```

But I've found a case of my own for what you said, and this is what I did (instead of letting the script break):

```bash
if [ -z "$(command -v rsync)" ]; then
  printf "\033[91m{package name} not installed! Check out {package webpage} or run the following command, then try again:\033[0m\n"
  printf "\033[95m{link to installation instructions}\033[0m\n"
fi
```

Anyway, I just figured that RSync comes built-in in most of the Unix-based systems, so you wouldn't need that message in the `README.md` at all, but that's totally fine to cover the case in the docs.

Btw, yours inspired me to build my own, so thanks!
